do not trigger action when element is hidden

just-boris opened the pull request:
It can be useful sometimes.
For example I have a set of tabs. One of tabs have a list with infinite scroll. Now scroll will be triggered anyway, even this tab is hidden. The fix resolves this issue and data will be loaded just when it's really needed

just-boris committed:
do not trigger action when element is hidden

just-boris commented:
Visibility check brought from jquery: https://github.com/jquery/jquery/blob/master/src/css/hiddenVisibleSelectors.js

JetFault commented:
+1 for this!

JetFault commented:
@just-boris While waiting for this, have you figured out a way to do this using only `infinite-scroll-disabled`?

just-boris commented:
@JetFault sometimes it is not possible because infitine-scroll doesn't know about visibility of parent nodes.
It happend when you put infinite scroll in angular-ui tabs: http://angular-ui.github.io/bootstrap/#/tabs, for example

just-boris commented:
P.S. I don't know why tests in saucelabs has been failed. On my local browser that worked fine.

kkirsche committed:
Remove moot `version` property from bower.json

Per bower/bower.json-spec@a325da3

Also their maintainer says they probably won't ever use it: http://stackoverflow.com/questions/24844901/bowers-bower-json-file-version-property
